card/card.gemspec

```diff
@@ -6,7 +6,7 @@ version = File.open(File.expand_path( '../VERSION', __FILE__ )).read.chomp
 Gem::Specification.new do |s|
   s.name          = 'card'
   s.version       = version
-  s.authors       = ["Ethan McCutchen", "Lewis Hoffman", "Gerry Gleason"]
+  s.authors       = ["Ethan McCutchen", "Lewis Hoffman", "Gerry Gleason", "Philipp Kühl"]
```

Comment:
Oh, this is the context.  Welcome to the list of major contributors, nice to have it growing.